.github/workflows/ci.yml

```diff
@@ -0,0 +1,64 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+env:
+  NODE_VERSION: 12
+
+jobs:
+  lint:
+    name: Lint files
+    runs-on: ubuntu-latest
+    timeout-minutes: 3
+    steps:
+      - name: Check out a copy of the repo
+        uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ env.NODE_VERSION }}
+        uses: actions/setup-node@v2-beta
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+
+      - name: Cache npm cache
+        id: cache-dependencies
+        uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-${{ env.NODE_VERSION }}-${{ secrets.CACHE_VERSION }}-npm-cache
+          restore-keys: ${{ runner.os }}-${{ env.NODE_VERSION }}-${{ secrets.CACHE_VERSION }}-
+
+      - name: Install dependencies
+        run: npm install alex@8.0.1 markdownlint-cli@0.19.0 --global
+        if: steps.cache-dependencies.outputs.cache-hit != 'true'
+
+      - name: Lint
+        run: |
+          ./scripts/validate.sh alex
+          ./scripts/validate.sh markdownlint
```

Comment:
If you tweak these scripts to use `npx markdown-cli` and whatnot, you won't need to install them globally (and they will always use the same version that **you** would use locally when running the linting scripts).

Reply:
I see. We do need a `package.json` to do this, is this correct?

I think updating the script may be better done by another person, in a separate pull request. (I haven't written a shell script in a while so I myself won't be able to update it quickly.)

Reply:
> I see. We do need a package.json to do this, is this correct?

No, I don't think we _need_ a `package.json` to do that, but I totally agree that we **should** have one.

Seems totally good to do separately, this does emulate the current `.travis.yml`. Maybe make an issue to migrate these shell scripts into `package.json` scripts (so we don't loose track)?

Reply:
Yep, sounds good.